### ISSUE TITLE
Feedback: Success with Gammy

### ISSUE THREAD

**Issue by Gitoffthelawn** [Fushko/gammy#103]
Hi Francesco,

I just wanted to take a moment to thank you for Gammy, and also provide you with the feedback that:

- Gammy works very well on Windows 7.  I haven't experienced any problems at all on Windows 7 boxes.
- Gammy works very well with an *Adaptation Speed* of zero.  This is actually my preferred way to use Gammy, because I find it is much more responsive when using this setting, which results in less eye strain.

Thank you again for the amazing tool!

**Comment by Fushko** [comment]
> Gammy works very well on Windows 7. I haven't experienced any problems at all on Windows 7 boxes.

Don't you get massive screen stutters when using it? I use GDI for getting screenshots, which is a really slow API. To be honest, I didn't think someone would find it bearable on this OS. Still, I'm glad it works for you.

**Comment by Gitoffthelawn** [comment]
It really works great for me.  No screen stutters at all.  I currently have the screenshot rate set at 800ms, but I haven't had any noticeable issues with any of the different values that I tried.  I was using 500ms for a while, and I might go back to that just because it's a little more responsive to brightness changes within applications.

Every once in a great while, I notice a tiny flicker.  I assume it's Gammy, but I've learned to ignore it because the benefit far outweighs the cost. :)
